### PR TITLE
changed CTA at the bottom of the page + stat about salary

### PR DESCRIPTION
**Pull request by rubenkostucki**

### CODE REVIEW

**Comment by nikeshashar** [comment]
@rubenkostucki could you post a screenshot of the change? (this is from a thought that I had about making changes to the site and helping someone squirrel quickly without having to run the PR locally).

File: source/employers.html.haml [hunk 1]
```diff
@@ -80,8 +80,8 @@
   .container
     .four-width-row.row-with-borders
       .column
-        %h2.statistic £30k
-        %h4.subheader Average Salary
+        %h2.statistic 10%
+        %h4.subheader Selected on the course
```
Comment (spike01) [inline source/employers.html.haml]: That doesn't make sense - I know what you're trying to say, but I can't think how to phrase it

Comment (spike01) [inline source/employers.html.haml]: 10% accepted on the course?

Comment (rubenkostucki) [inline source/employers.html.haml]: @spike01 - what about

        %h2.statistic 30
        %h4.subheader Developers every 6 week

**Pull request closed without merging**